spec/runners/crud/requirement.rb

```diff
@@ -42,7 +42,7 @@ def satisfied?
           ok &&= cc.fcv_ish >= short_min_server_version
         end
         if max_server_version
-          ok &&= cc.server_version <= max_server_version
+          ok &&= cc.short_server_version <= max_server_version
```

Comment:
I changed this to be consistent with how the CRUD spec runner decides whether to run or skip tests: [https://github.com/mongodb/mongo-ruby-driver/blob/master/spec/runners/crud.rb#L209](https://github.com/mongodb/mongo-ruby-driver/blob/master/spec/runners/crud.rb#L209)